### ISSUE TITLE
Averages Might Not Be the Best Measure to Display

### ISSUE THREAD

**Issue by BrianKang123** [btieu30/emp_ABA#2]
Your experiment introduces too many different cases. Although you do run the searches 10 different times, statistically half of the arrays will have worst case scenarios, not to mention each array is a different case. You are then taking the overall average of all of these experiments. It would be best, for your case, to take the average of 10 experiments of the same array length, then doing that same process 10 more times for arrays of different lengths, or just don't take the average, due to the results being skewed. 

TLDR: Test and collect data of one variable at a time.  
-AllBrians
